packages/util/src/object.ts

```diff
@@ -5,11 +5,13 @@ const proxyIdentitySymbol = Symbol('[[proxyIdent]]');
 const proxyTargetSymbol = Symbol('[[proxyTarget]]');
 
 export type PropertyTransformer<T> = (target: T, name: string | number | symbol) => string | number | symbol | undefined;
+export type ValueTransformer = (value: string | number | boolean | undefined) => any;
 
 export class PropertyTransformHandler<T extends object> implements ProxyHandler<T> {
 
     constructor(
         private readonly transformProperty: PropertyTransformer<T>,
+        private readonly transformValue: ValueTransformer = (value) => value,
```

Comment:
There is a unit test (packages/util/src/__tests__/transformProxy.test.ts) for this class, please update it so that value transformer functionality can be verified.